.eslintrc

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
I think the convention is now to name the file .eslintrc.json instead of .eslintrc